doc/users/next_whats_new/2017_11_15_datetime64.rst

```diff
@@ -0,0 +1,10 @@
+Support for numpy.datetime64
+----------------------------
+
+Matplotlib has supported `datetime.datetime` dates for a long time.  We
+now support `numpy.datetime64` dates as well.  This should be
+transparent to the user, and anywhere that `dateime.datetime` could be
```

Comment:
I think "this should be transparent to the user" falls into the "this is simple/obvious/easy" ableist type language that I think should be avoided when possible.

Reply:
I just meant they shouldn't have to do anything special to make it happen.  i.e. there is no new API, they just use `plot` or `date2num` as they always have.

Reply:
I think it might be clearer to say that? np.datetime is now a supported type, so np.datetime objects can now be used as data inputs to any plotting function that already supports dates.